frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -105,10 +105,12 @@ class BitBox02 extends Component<Props, State> {
             agreement4: false,
             agreement5: false,
             waitDialog: undefined,
+            passwordEntryLoaded: false,
         };
     }
 
     private unsubscribe!: () => void;
+    private PasswordEntry: any | null = null;
```

Comment:
why not put this into the state? then you don't need two vars.

Reply:
also why `any  | null`? `any` includes `null`. And in TS `undefined` is preferred over `null` :joy:

Reply:
That would have been IPasswordEntry interface + null if we would have gone with dynamically loading the module, see my comment

> I'll remove the dynamic loading although I would have liked to figure out how to properly do dynamic module loading with correct typings.

https://github.com/digitalbitbox/bitbox-wallet-app/pull/1416#issuecomment-894135231